opm/grid/CpGrid.hpp

```diff
@@ -262,6 +262,12 @@ namespace Dune
         /// \param turn_normals if true, all normals will be turned. This is intended for handling inputs with wrong orientations.
         /// \param clip_z if true, the grid will be clipped so that the top and bottom will be planar.
         /// \param poreVolume pore volumes for use in MINPV processing, if asked for in deck
+        ///
+        /// The return value is a list of global indices to the cells which have
```

Comment:
Any reason not to use `/// \return a list ,,,`

Reply:
No reason - apart from ignorance; updated now.